packages-exp/auth-exp/src/platform_browser/persistence/local_storage.test.ts

```diff
@@ -143,6 +143,22 @@ describe('platform_browser/persistence/local_storage', () => {
           expect(callback).to.have.been.calledWith(newValue);
         });
 
+        it('should trigger on storage event for the same key after local change', async () => {
+          await persistence._set(key, newValue);
+          await new Promise(resolve => setTimeout(resolve, 100)); // wait after the event trigger
+          callback.resetHistory();
+
+          window.dispatchEvent(
+            new StorageEvent('storage', {
+              key,
+              oldValue: JSON.stringify(newValue),
+              newValue: null
+            })
+          );
+
+          expect(callback).to.have.been.calledOnceWith(null);
```

Comment:
This test won't pass and the error message isn't helpful:

```
platform_browser/persistence/local_storage
    #addEventListener
      with events
        with a change in the underlying storage
          ✖ should trigger on storage event for the same key after local change
            Chrome Headless 89.0.4389.90 (Mac OS 11.2.3)
          AssertionError: expected spy to have been called exactly once with arguments null

      + expected - actual

      
    at src/platform_browser/persistence/local_storage.test.js:38085:70
```

Reply:
Maybe try breaking it into two expects? Switch to `to.have.been.called.with(null)` and `to.have.been.called.once`

Reply:
Thanks, `calledOnce` gives a more reasonable message `AssertionError: expected spy to have been called exactly once, but it was called 0 times`. Now I still need to figure out why